### PR TITLE
Entgo and Sveltekit Sample Project

### PR DESCRIPTION
**Pull request by ugurkorkmaz**
Hello, I have developed a sample project for Entgo (ORM) and SvelteKit. Could you please review my pull request?

### CODE REVIEW

File: entgo-sveltekit/template/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (gaby) [inline entgo-sveltekit/template/package.json:1]: This file has to be added to .github/dependabot.yml

File: entgo-sveltekit/go.mod [hunk 2]
```diff
@@ -0,0 +1,56 @@
+module app
```
Comment (gaby) [inline entgo-sveltekit/go.mod:1]: This one also needs to be added to .github/dependabot.yml

File: entgo-sveltekit/go.mod [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+go 1.20
```
Comment (gaby) [inline entgo-sveltekit/go.mod:3]: Use go 1.19 here

**Review by gaby — CHANGES_REQUESTED** [review]

File: entgo-sveltekit/template/package.json (hunk 1 above)
Comment (ugurkorkmaz) [reply entgo-sveltekit/template/package.json:1]: I made the fix.

File: entgo-sveltekit/go.mod (hunk 2 above)
Comment (ugurkorkmaz) [reply entgo-sveltekit/go.mod:1]: I made the fix.

File: entgo-sveltekit/go.mod (hunk 3 above)
Comment (ugurkorkmaz) [reply entgo-sveltekit/go.mod]: I switched to version 1.19

**Comment by ugurkorkmaz** [comment]
The bot is throwing an error because the 'dist' folder is included in the .gitignore. What can we do? Can I send it to the repository? @gaby

**Comment by gaby** [comment]
@ugurkorkmaz Try adding this to the files hound bot is complaining about:

```
// jshint esversion:6
```

**Comment by gaby** [comment]
@ugurkorkmaz Awesome!

**Review by gaby — APPROVED** [review]
LGTM

**Review by efectn — APPROVED** [review]

**Comment by gaby** [comment]
@ReneWerner87 thoughts?

**Comment by ReneWerner87** [comment]
Oh i missed it, will merge

**Comment by ugurkorkmaz** [comment]
@ReneWerner87 @gaby @efectn Thanks.

**Pull request merged**